### PR TITLE
Alpha note, and recommendations on pinning versions

### PR DESCRIPTION
**Pull request by tomchristie**
Screenshot 2019-12-04 at 10 02 37

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -30,9 +30,16 @@ It can be used in high-performance async web frameworks, using either asyncio
 or trio, and is able to support making large numbers of concurrent requests.
 
 !!! note
-    The 0.8 release switched HTTPX into focusing exclusively on providing an async
-    client. It is possible that we'll look at re-introducing a sync API at a
-    later date.
+    HTTPX should currently be considered in alpha. We'd love early users and feedback,
+    but would strongly recommend pinning your dependencies to the latest median
+    release, so that you're able to properly review API changes between package
+    updates. Currently you should be using `http==0.8.*`.
```
Comment (florimondmanca) [inline docs/index.md]: Small typo here: `httpx`

File: docs/index.md [hunk 2]
```diff
@@ -30,9 +30,16 @@ It can be used in high-performance async web frameworks, using either asyncio
...
+
+    In particular, the 0.8 release switched HTTPX into focusing exclusively on
+    providing an async client, in order to move the project forward, and help
+    us [change our approach to providing sync+async support][sync-support]. If
+    you have been using the sync client, you may want to pin to `http==0.7.*`,
```
Comment (florimondmanca) [inline docs/index.md]: Same typo in here :)

File: docs/index.md (hunk 1 above)
Comment (florimondmanca) [reply docs/index.md]: And actually these also apply to the repo README.

Comment (tomchristie) [reply docs/index.md]: 😬 *Thank you!*

**Pull request merged**